src/components/cylc/MessageChip.vue

```diff
@@ -0,0 +1,56 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<template>
+  <div>
+    <v-tooltip :activator="null">
+      <template v-slot:activator="{ props }">
+        <v-chip
+          v-bind="props"
+          :class="chipClass"
+          class="ml-2 message-output"
+          size="small"
+        >
+          {{ level }}
+        </v-chip>
+      </template>
+      <span>{{ message }}</span>
+    </v-tooltip>
+  </div>
+</template>
+
+<script setup>
+import { computed } from 'vue'
+
+const props = defineProps({
+  level: String,
+  message: String,
+  isMessage: Boolean
+})
+
+const classMap = new Map([
+  ['this is a debug message', ''],
+  ['this is a info message', 'bg-grey'],
+  ['this is a warning message', 'bg-warning'],
+  ['this is an error message', 'bg-error'],
+  ['this is a critical message', 'bg-black font-weight-bold'],
```

Comment:
This won't do anything until https://github.com/cylc/cylc-flow/issues/5713 is addressed, but this is how it should be
```suggestion
  ['DEBUG', ''],
  ['INFO', 'bg-grey'],
  ['WARNING', 'bg-warning'],
  ['ERROR', 'bg-error'],
  ['CRITICAL', 'bg-black font-weight-bold'],
```

Can you take a look at

https://github.com/cylc/cylc-ui/blob/63bdcc2c231e0338459fe335391ea2f6adfbbf6d/src/utils/tasks.js#L70

and add some logic to extract the severity level from the message which will be in the form `SEVERITY:rest of message`

Reply:
(I've got a fix for the "output vs ordinary message" logic at #1456)